composer.json

```diff
@@ -11,8 +11,8 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "symfony/http-foundation": "~2.1",
-        "symfony/http-kernel": "~2.1"
+        "symfony/http-foundation": "~2.1|~3.0|~4.0",
+        "symfony/http-kernel": "~2.1|~3.0|~4.0"
```

Comment:
What about bumping `~2.1` to `~2.7`? `2.1` is very old now, so I don't think we need to support it.